src/css/style.css

```diff
@@ -1,5 +1,255 @@
+}
+
+/* Styling the inverted greater-than sign */
+.arrow::before {
+    content: "\203A"; /* Unicode character for an inverted greater-than sign */
+    display: inline-block;
+    transform: translateY(2px); /* Adjusting vertical alignment */
+    transition: transform 0.3s; /* Adding smooth transition for arrow rotation */
+}
+
+/* Rotating the arrow when the "sol" class is hovered */
+.sol:hover .arrow::before {
+    transform: translateY(2px) rotate(180deg); /* Rotating the arrow 180 degrees on hover */
+}
+
+/* Optionally, customize arrow color or other styles */
+.arrow::before {
+    color: #000; /* Arrow color (adjust to your preference) */
+    font-size: 14px; /* Arrow size (adjust if needed) */
+}
+
+
+/* Style the "Contact Us" button */
+.button {
+    background: #2d2e52;
+    border-radius: 22px;
+    width: 8.375rem;
+    padding: 0.7em;
+    font-weight: bold;
+    font-family: Arial, sans-serif;
+}
+
+.contact-us {
+    background: #2d2e52;
+    border-radius: 22px;
+    width: 8.375rem;
+    padding: 0.7em;
+    border: none;
+    text-align: center;
+    color: #fff;
+    cursor: pointer;
+    margin-left: auto; /* Push "Contact Us' to the right by using auto margin */
+}
+
+/* Add styles for the mobile menu */
+.nav-toggle {
+    display: none;
+}
+
+.nav-menu {
+    display: flex; /* Change to flex to make navigation items horizontal */
+    align-items: center; /* Center navigation links vertically */
+}
+
+/* Styles for the hamburger menu */
+.hamburger {
+    display: none;
+    cursor: pointer;
+}
+
+.line {
+    width: 30px;
+    height: 3px;
+    background-color: #000; /* Change the color as needed */
+    margin: 4px 0;
+}
+
+/* Add styles for the contact button inside the menu */
+.contact-us-mobile {
+    background: #2d2e52;
+    border-radius: 22px;
+    width: 8.375rem;
+    padding: 0.7em;
+    border: none;
+    text-align: center;
+    color: #fff;
+    cursor: pointer;
+    display: none; /* Hide the mobile "Contact Us" button by default */
+    margin-top: 10px; /* Add spacing between the button and menu items */
+
+}
+
+.contact-button1 {
+    display: none;
+    color: white;
+}
+
+
+.contact-us {
+    display: inline-block; /* Show the "Contact Us" button */
+    margin-top: 10px; /* Add spacing between the button and menu items */
+
+}
+
+/* Add a media query to switch to the hamburger menu on smaller screens */
+@media (max-width: 768px) {
+    .nav-container {
+        text-align: center; /* Align navigation buttons in the center */
+    }
+
+    .nav-menu {
+        display: none;
+        flex-direction: column; /* Stack navigation items vertically */
+        background-color: #fff; /* Background color for the mobile menu */
+        position: absolute; /* Position the mobile menu */
+        top: 100%; /* Position it below the header */
+        left: 0;
+        right: 0;
+        z-index: 1000; /* Ensure it's above other content */
+    }
```

Comment:
display is none, other styles not necessary. Please attach screenshot after the changes.